geom/geojson_feature_collection.go

```diff
@@ -153,7 +153,7 @@ func (c *GeoJSONFeatureCollection) UnmarshalJSON(p []byte) error {
 		Type     string           `json:"type"`
 		Features []GeoJSONFeature `json:"features"`
 	}
-	if err := json.Unmarshal(p, &topLevel); err != nil {
+	if err := json.Unmarshal(p, &topLevel); err != nil { //nolint:musttag
```

Comment:
I believe there's a false positive bug in `musttag` that's triggered by this code.

I will try to create a minimum reproduction of the problem and, if successful, will raise a bug report.

Reply:
Reported here: https://github.com/go-simpler/musttag/issues/93